themes/bootstrap3/js/record.js

```diff
@@ -34,6 +34,37 @@ function checkRequestIsValid(element, requestType, icon = 'place-hold') {
     });
 }
 
+function wayfinderPlacementLinkLookup(element) {
+  var url = VuFind.path + '/AJAX/JSON?' + $.param({
```

Comment:
Any new JS code should avoid jQuery. Use `fetch`, `URLSearchParams`, `document.querySelectorAll` etc. instead. See e.g. finna-organisation-info.js for examples. And let my know if you need more assistance!